openpype/tools/launcher/models.py

```diff
@@ -222,6 +241,75 @@ def filter_compatible_actions(self, actions):
             key=lambda action: (action.order, action.name)
         )
 
+    def update_force_not_open_workfile_settings(self, is_checked, action):
+        """Store/remove config for forcing to skip opening last workfile.
+
+        Args:
+            is_checked (bool): True to add, False to remove
+            action (ApplicationAction)
+        """
+        actual_data = self._prepare_compare_data(action)
```

Comment:
```suggestion
    def update_force_not_open_workfile_settings(self, is_checked, item_id):
        """Store/remove config for forcing to skip opening last workfile.

        Args:
            is_checked (bool): True to add, False to remove
            action (ApplicationAction)
        """
        item = self.items_by_id.get(item_id)
        if not item:
            return
        action = item.data(ACTION_ROLE)
        actual_data = self._prepare_compare_data(action)
```

And on the end of method trigger data change to trigger repaint
```
self.dataChanged.emit(item.index(), index, [QtCore.Qt.DisplayRole])
```